server/sockets/utilityFunctions.js

```diff
@@ -0,0 +1,21 @@
+// socket utility functions
+const db = require('../../database/database.js');
+
+
+module.exports = {
+  dropNCreateDBTables: () => {
+    db.complex.dropAllTables()
+    .then(db.complex.initializeDB)
+    .catch(err => console.error('Error dropping or creating tables', err));
```

Comment:
The function to drop and create the database still runs when the server starts up, since it has to run the whole file on being required by the sockets/path file, which in turn is required by the server/index file, you make a very good point that this is an odd place to run this database initialization step. I'll refactor a bit to require the utility functions on the server as well, and drop and create the db tables in the index file 👍